docs/synctl-create-test.md

```diff
@@ -44,21 +45,18 @@ synctl create test [options]
 ```
 ### Options for API Script test
 ```  
-    -f, --from-file <file>              load synthetic test payload from file (.json)
     --script <file>                     load script (.js) from file
     --bundle <bundle>                   Synthetic bundle test script, support zip file, zip file encoded with base64
     --bundle-entry-file <file-name>     Synthetic bundle test entry file, e.g, myscript.js
 ```
 ### Options for Browser Script test
 ```    
-    -f, --from-file <file>              load synthetic test payload from file (.json)
     --script <file>                     load script (.js) from file
     --browser <string>                  browser type, support chrome and firefox
     --record-video <boolean>            set true to record video
 ```
```

Comment:
"--bundle" and   " --bundle-entry-file" should be also available for browser script test

--bundle <bundle>                   Synthetic bundle test script, support zip file, zip file encoded with base64
    --bundle-entry-file <file-name>